Northstar.Client/mod/scripts/vscripts/spectator_pov_button_fix.nut

```diff
@@ -0,0 +1,12 @@
+global function SpectatorPovButtonFixInit
+
+void function SpectatorPovButtonFixInit(){
+	RegisterButtonPressedCallback(MOUSE_MIDDLE, SwitchPov)
```

Comment:
The button callback needs to be only registered when spectating

Reply:
Registering and deregistering it would be pain, especially because all of the possible combinations of spectating/respawning/joining a game/switching players etc., so unless I manage do find proper callbacks for that I'd leave it as it is, since it only does something when spectating